packages/destination-actions/src/destinations/fullstory/request-params.ts

```diff
@@ -124,13 +124,13 @@ export const setUserPropertiesRequestParams = (
 }
 
 /**
- * Returns {@link RequestParams} for the delete user HTTP API endpoint.
+ * Returns {@link RequestParams} for the V2 delete user HTTP API endpoint.
  *
  * @param settings Settings configured for the cloud mode destination.
  * @param userId The id of the user to delete.
  */
 export const deleteUserRequestParams = (settings: Settings, userId: string): RequestParams => {
-  const defaultParams = defaultRequestParams(settings, `users/v1/individual/${encodeURIComponent(userId)}`)
+  const defaultParams = defaultRequestParams(settings, `v2beta/users?uid=${encodeURIComponent(userId)}`)
```

Comment:
Are we comfortable making this change immediately on deploy for all users? This change isn't gated behind a new opt-in action like our other V2 work so far.

Reply:
This is a tough one. From what I can tell there is no way to leave V1 in place and make V2 an option.  There is only the [onDelete](https://github.com/cowpaths/segmentio-action-destinations/blob/0474006c045de81647afa0c6e87e3884489244ab/packages/destination-actions/src/destinations/fullstory/index.ts#L43) function on the Destination Definition. I cannot find how this is invoked. Is it done automatically based on setting up the destination action and data being removed from segment?
cc: @patrick-fs

Reply:
The good news is that the `POST /delete` behavior between API V1 and V2 is identical.

The bad news, I think, is that `v2beta` requires an Admin key and `v1` can use a standard key. @mattgrosvenor-fs do I have that right?

Reply:
> I cannot find how this is invoked. Is it done automatically based on setting up the destination action and data being removed from segment?

That's my understanding -- that this is triggered automatically if the user is deleted in Segment.  From their docs:

```
  onDelete: async (request, { settings, payload }) => {
    // Return a request that performs a GDPR delete for the provided Segment userId or anonymousId
    // provided in the payload. If your destination does not support GDPR deletion you should not
    // implement this function and should remove it completely.
  },
```

Reply:
@patrick-fs that is correct.

Reply:
[Related convo](https://fullstory.slack.com/archives/CN268EB8C/p1689191912223769) happening in #proj-api about required permissions of API keys, but I would say we should *not* move forward with this change to delete users if it suddenly requires an Admin key. We risk breaking existing integrations.

Reply:
@mattgrosvenor-fs @joshrynneFS it looks like the change was merged - It might need to be reverted, then, if we decide that `DELETE /v2beta/users` will continue to require an admin key